taker-frontend/src/components/NavBar.tsx

```diff
@@ -20,9 +21,10 @@ import { WalletInfo } from "./Types";
 
 interface NavProps {
     walletInfo: WalletInfo | null;
+    makerStatus: boolean;
```

Comment:
If this is a boolean, I would name it `connectedToMaker`.

Reply:
renamed it everywhere (most of the places had `isMakerOnline`, but I don't mind your version.